libcontainer/configs/cgroup.go

```diff
@@ -57,6 +57,11 @@ type Cgroup struct {
 	// MEM to use
 	CpusetMems string `json:"cpuset_mems"`
 
+	// Process limit; set to `0' to disable limit.
+	// While technically `0' is a valid PID limit, it does not make sense in the
+	// context of a container -- it is identical to a limit of `1'.
```

Comment:
@cyphar Because pids.max = 0 is also working equally with pids.max = 1 in pids cgroup.  When there is no process in cgroup, both of them would not work, when there is any process in cgroup, both of them are preventing user to fork() or clone().